generate_version_header.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+VERSION_TEMPLATE="#pragma once
+namespace principia {
+    namespace base {
```

Comment:
Although it's a generated file that (presumably) no-one will look at, it would be good if it were indented like the rest of the code, thus:

```
namespace principia {
namespace base {
char const kBuildDate[] = \"%%DATE%%\";
char const kVersion[] = ...
}  // namespace base
}  // namespace principia